esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -395,7 +433,9 @@ void setup()
 
     // Serial 
     size_t rxbufsize = SERIAL.setRxBufferSize(2*1024); // must come before uart started, retuns 0 if it fails
+#ifndef ESP8266 // do we even need this on esp32?
```

Comment:
why this change?

Reply:
Because esp8266 does not define serial.setTxBufferSize()

Reply:
so pl change the comment to // not supported by ESP8266

Reply:
I see you have merged,  I'd suggest we seek an answer to the question I asked in this comment as google suggests that it values over the default of 128 may not actually do anything.  If it's not needed, we can remove one ifdef.   I hope you will find time to look at the other two much simpler PRs, when they are merged, I can work on my UDPCl ideas.  Thanks!